spec/test/coverage_spec.rb

```diff
@@ -13,6 +13,7 @@
   after(:each) do
     described_class.uninstall
     described_class.reset
+    OpenapiFirst::Test.definitions.clear
```

Comment:
`OpenapiFirst::Test.definitions.clear` is already called globally after each test. [See spec_helper.rb](https://github.com/ahx/openapi_first/blob/main/spec/spec_helper.rb#L31). This this should not be necessary here.